.gitignore

```diff
@@ -18,3 +18,6 @@ Gemfile.lock
 .feature_map/metrics.yml
 .feature_map/test-coverage.yml
 .feature_map/docs
+
+# Gem build artifacts
+feature_map-*.gem
```

Comment:
Running `gem build feature_map.gemspec` to deploy a new version of this gem creates a local build artifact for the gem that we don't want to be accidentally committed.